mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6147,6 +6145,21 @@ func containerClusterAutopilotCustomizeDiff(_ context.Context, d *schema.Resourc
 	return nil
 }
 
+// Default to VPC_NATIVE mode during initial creation
```

Comment:
I think this is reasonable _for GKE cluster_ but not necessarily generally. Suggested a blurb.

```suggestion
// Default to VPC_NATIVE mode during initial creation
// This solution (a conditional default) should not be considered to set a precedent on its own.
// If you're considering a similar approach on a different resource, strongly weigh making the field required.
// GKE tends to require exceptional handling in general- and this default was a breaking change in their API
// that was compounded on by numerous product developments afterwards. We have not seen a similar case
// since, after several years.
```